### DISCUSSION TITLE
Reduce Members in Ensemble API?

### DISCUSSION THREAD

**Discussion by TSellers2** [open-meteo/open-meteo#592] [Q&A]
When I first used this, the chart displayed one line for each model, now I see numerous members for each model (GEM, GFS, and ECMWF). Is there a way to get it back to just one line for each model?

**Comment by guidocioni** [comment]
> When I first used this, the chart displayed one line for each model, now I see numerous members for each model (GEM, GFS, and ECMWF). Is there a way to get it back to just one line for each model?

If you just want to get 1 member in ensemble forecasts you'd rather use directly deterministic forecasts (forecast API). 
The added value of ensemble data is exactly having more realizations for the same variables.

**Comment by TSellers2** [comment]
Thanks for that. My intent is to use the API to compare forecast values
between models out to 7 days, I am under the impression I need to use the
Ensemble API for that?

Regards,
Tom Sellers

**Reply by guidocioni** [reply]
Nope. if your goal is only that then I'll suggest you to use the forecast API where you can compare more models than the ensemble with higher resolutions and more available parameters

**Comment by TSellers2** [comment]
Thanks for that.

I'm making up a video that compares the GEM, GFS, and ECMWF forecast models
out to 7 days.

For some reason when I ran the Ensemble API with GFS and GEM on a Linux
machine (using Chrome), it returned one member for each. Then I ran it
again on a windows machine (also Chrome) and I got 2 or 3 pages the
members. It seemed that the single line was a more visual way to compare
because it had the advantage of simplicity for the viewer to follow, (and
the ensemble API allows you to use the vertical ruler to compare values at
the same time). That's what lead to my assumption it may be possible.

Regards,
Tom

**Reply by patrick-zippenfenig** [reply]
Let us know how the video turns out! As @guidocioni mentioned, make sure to use the regular [forecast API](https://open-meteo.com/en/docs/) and select models from the "Weather models" dropdown

**Comment by TSellers2** [comment]
I also seem to encounter another problem with API Response if I refresh the
browser and then 'reload chart'. I get the message "Can not preview more
than 5 locations".

Regards,
Tom

**Reply by patrick-zippenfenig** [reply]
That's an issue of the API documentation. The location is stored in the URL hash and if the page is reloaded, it screws up the format. If you select a location again, it works.  I created a new issue ticket here: https://github.com/open-meteo/open-meteo-website/issues/145

**Comment by TSellers2** [comment]
Thanks for all the responses, that will help a lot.

The point of the video is to compare the result of infrastructure cuts for
ten years (the GEM, which can only use hydrostatic modeling) with other
agencies that can afford supercomputers.

I'll be sure to send a link when it is up.

Regards,
Tom
